server/pulp/server/webservices/views/content.py

```diff
@@ -535,21 +557,6 @@ def post(self, request, action):
         else:
             return HttpResponseBadRequest('bad request')
 
-    def refresh(self, request):
-        """
-        Refresh all content sources
-
-        :param request:   WSGI request object, body contains bits to upload
-        :type  request:   django.core.handlers.wsgi.WSGIRequest
-        :raises: OperationPostponed when an async operation is performed
-        """
-        container = ContentContainer()
-        content_sources = [tags.resource_tag(RESOURCE_CONTENT_SOURCE, id)
-                           for id in container.sources.keys()]
-        task_tags = [tags.action_tag(ACTION_REFRESH_ALL_CONTENT_SOURCES)] + content_sources
-        task_result = content.refresh_content_sources.apply_async(tags=task_tags)
-        raise OperationPostponed(task_result)
-
 
 class ContentSourceResourceView(View):
```

Comment:
I think you should split this view into ContentSourceResourceView and ContentSourceResourceActionView overwise you will get error while doing http://fpaste.org/220591/34127014/